ocean_provider/run.py

```diff
@@ -25,11 +25,30 @@ def get_version():
 
 @app.route("/")
 def version():
+    """
+    Contains the provider data for an user:
+        - software;
+        - version;
+        - network url;
+        - provider address;
+        - services endpoints, which has all
+        the existing endpoints from route.py.
+    """
     info = dict()
     info['software'] = Metadata.TITLE
     info['version'] = get_version()
     info['network-url'] = config.network_url
     info['provider-address'] = get_provider_wallet().address
+    info['servicesEndpoints'] = {
+        'access': url_for('services.download'),
+        'compute': url_for('services.compute_get_status_job'),
```

Comment:
maybe call this `job_status`

Reply:
if we generate the dict dynamically, then we can rename the function names to reflect what we want in the dict keys.

Reply:
At my first PR, I kept just the job_status function, because all functions related to 'compute' service have the exact same route and that's why I didn't include them. Now, I saw that it's better to have a dynamic dict, so I will refactor this.